.travis.yml

```diff
@@ -10,5 +11,8 @@ env:
 script:
   - make test config=coverage defines=$DEFINES -j2
   - make test config=release defines=$DEFINES -j2
+  - make test cxxstd=c++98 -j2
+  - make test cxxstd=c++03 -j2
+  - make test cxxstd=c++11 -j2
 
```

Comment:
I think it's a good idea to have a test for c++98; the other two are probably redundant since c++11 can be enabled by default (you can spell it out directly in the first two steps) and c++03 is a weird thing to test for in general. That'd limit the build time degradation to just having 1 extra step.